package.json

```diff
@@ -5,6 +5,7 @@
   "description": "JavaScript library for mobile-friendly interactive maps",
   "devDependencies": {
     "@rollup/plugin-json": "^4.1.0",
+    "bundlesize": "0.18.1",
```

Comment:
Can you use a version range here instead of an exact version? We're using a lockfile, which takes care of locking the versions for us.